src/main/java/uk/gov/companieshouse/account/validator/controller/AccountValidationController.java

```diff
@@ -173,9 +173,8 @@ public String getFilename() {
             }
         };
 
-        MultiValueMap<String, Object> map = new LinkedMultiValueMap<>() {{
-            add("instance", contentsAsResource);
-        }};
+        MultiValueMap<String, Object> map = new LinkedMultiValueMap<>();
+        map.add("instance", contentsAsResource);
```

Comment:
Previous format was a non-compliant initialisation shown here:
https://code-analysis.platform.aws.chdev.org/project/issues?id=uk.gov.companieshouse%3Aaccount-validator-api&issues=AYdRsm0x1r4uIxLEJyWg&open=AYdRsm0x1r4uIxLEJyWg

This has now been changed to match compliant solution